source/_docs/automation/trigger.markdown

```diff
@@ -763,7 +763,7 @@ automation 3:
   trigger:
     - platform: time_pattern
       # You can also match on interval. This will match every 5 minutes
-      minutes: "/5"
+      minutes: /5
```

Comment:
This change is incorrect. Time pattern is a string and, therefore, should be enquoted.